plugins/modules/setup.yml

```
@@ -55,6 +55,6 @@ EXAMPLES: |
   - name: run the setup facts
     ansible.windows.setup:
 
-   - name: Gather all facts with a custom timeout on Windows host
-     ansible.windows.setup:
-       gather_timeout: 30 
+  - name: Gather all facts with a custom timeout on Windows host
+    ansible.windows.setup:
+      gather_timeout: 30 
```

```suggestion
      gather_timeout: 30
```

Looks like there was also a trailing space as well here.